src/common/TableOfContents.jsx

```diff
@@ -67,7 +69,7 @@ const TableOfContents = ({
   return (
     <div>
       <div className='toc-container'>
-        <BackLink year={year} />
+        {displayTOCBackLink == true ? <BackLink year={year} /> : ''}
```

Comment:
BackLink has a `hide` prop that can be used to conditionally render it.
```suggestion
        <BackLink year={year} hide={!displayTOCBackLink} />}
```

Reply:
Has been updated!